File: 2678-number-of-senior-citizens/2678-number-of-senior-citizens.java
class Solution {
    public int countSeniors(String[] details) {
        int count=0;
        for(int i=0;i<details.length;i++){
            String k=details[i].substring(11);
            int l=Integer.valueOf(k);
            l/=100;
            if(l>60)count++;
        }
        return count;
    }
}

File: 2678-number-of-senior-citizens/2678-number-of-senior-citizens.c
int countSeniors(char ** details, int detailsSize){
    int count=0;
    for(int i=0;i<detailsSize;i++){
        int a=details[i][11]-'0';
        int b=details[i][12]-'0';
        int num=a*10+b;
        if(num>60)count++;
    }
    return count;
}